source/features/git-checkout-pr.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import ClippyIcon from 'octicon/clippy.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCurrentBranch} from '../github-helpers';
+
+function handleMenuOpening({delegateTarget: dropdown}: delegate.Event): void {
+	dropdown.classList.add('rgh-git-checkout'); // Mark this as processed
+	const [, user, repository] = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname.split('/', 3);
```

Comment:
Should this be put in its own file and exported as prUser, prRepository?

Reply:
Is it used elsewhere?

Reply:
https://github.com/sindresorhus/refined-github/search?q=const+%5B%2C+user%2C+repository%5D+%3D+select%3CHTMLAnchorElement%3E%28%27.commit-ref.head-ref+a%27%29%21.pathname.split%28%27%2F%27%2C+3%29%3B

Reply:
Ill do it in a lint later